serverless/containers/index.mdx

```diff
@@ -2,7 +2,93 @@
 meta:
   title: Serverless Containers Documentation
   description: Serverless Containers Documentation
-content:
-  h1: Containers Documentation
-  paragraph: Containers Documentation
 ---
+
+<ProductHeader 
+   productName="Serverless Containers"
+   productLogo="containersWire"
+   description="Scaleway Serverless Containers allows you to deploy containerized stateless web applications in the cloud to run short-lived, event-driven tasks."
+   url="/serverless/containers/quickstart"
+   label="Serverless Containers Quickstart"
+/>
+
+## Getting Started
+
+<Grid>
+    <SummaryCard
+        title="Quickstart"
+        icon="rocket"
+        description="Learn how to create and deploy a Serverless Container in a few steps."
+        label="View Quickstart"
+        url="/serverless/containers/quickstart/"
+    />
+    <SummaryCard
+        title="Concepts"
+        icon="info"
+        description="Core concepts that give you a better understanding of Serverless."
+        label="View Concepts"
+        url="/serverless/containers/concepts/"
+    />
+    <SummaryCard
+        title="How-Tos"
+        icon="help-circle-outline"
+        description="Learn how to use Serverless Containers using the Scaleway console."
+        label="View How-Tos"
+        url="/serverless/containers/how-to/"
+    />
+    <SummaryCard
+        title="Additional resources"
+        icon="book-open-outline"
+        description="Go further with detailed, in-depth information on Serverless Containers."
+        label="View Additional resources"
+        url="/serverless/containers/reference-content/"
+    />
+</Grid>
+
+## Tutorials
+
+<Grid>
+    <DefaultCard
+        title="Running a Python Flask Server on a Serverless Container"
+        url="tutorials/run-python-flask-server-on-serverless-container/"
+        label="Read more"
+    />
+    <DefaultCard
+        title="Running a Node.js + Express Server on Serverless Container"
+        url="tutorials/run-nodejs-express-server-on-serverless-containers/"
+        label="Read more"
+    />
+    <DefaultCard
+        title="Deploying Umami web analytics on Serverless Containers"
+        url="tutorials/deploy-umami-on-serverless-containers/"
+        label="Read more"
+    />
+    <DefaultCard
+        title="Deploying Laravel 10.x on Serverless Containers"
+        url="tutorials/deploy-laravel-on-serverless-containers/"
+        label="Read more"
+    />
+</Grid>
+
+
+<ClickableBanner
+    productLogo="cliWire"
+    title="Serverless Containers API"
+    description="Manage Serverless Containers using the Scaleway API."
+    url="https://www.scaleway.com/en/developers/api/serverless-containers/"
+    label="Go to Serverless Containers API"
+/>
+
+## Changelog
+
+<ChangelogList
+    productName="Serverless Containers"
+    numberOfChanges={3}
+/>
+
+<ClickableBanner
```

Comment:
you have to remove this, bc they added a banner that is added automatically. otherwise there'll be duplicates